physical/postgresql/postgresql.go

```diff
@@ -259,43 +259,8 @@ func NewPostgreSQLBackend(conf map[string]string, logger log.Logger) (physical.B
 		return nil, fmt.Errorf("failed to parse value for `skip_create_table`: %w", err)
 	}
 	if !skip_create_table {
-		txn, err := db.BeginTx(context.TODO(), &sql.TxOptions{
-			Isolation: sql.LevelRepeatableRead,
-		})
-		if err != nil {
-			return nil, fmt.Errorf("failed to begin transaction to attempt table creation: %w", err)
-		}
-
-		defer txn.Rollback()
-
-		createTableQuery := "CREATE TABLE IF NOT EXISTS " + quoted_table + " (" +
-			`parent_path TEXT COLLATE "C" NOT NULL,` +
-			`  path        TEXT COLLATE "C",` +
-			`  key         TEXT COLLATE "C",` +
-			`  value       BYTEA,` +
-			`  CONSTRAINT pkey PRIMARY KEY (path, key)` +
-			`);`
-		if _, err := db.Exec(createTableQuery); err != nil {
```

Comment:
@wslabosz-reply I forgot to point out too, that `txn` was entirely unused and we executed the creation directly.